.github/workflows/tests-cli.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     - uses: actions/setup-node@v1
       with:
         node-version: '10.x'
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v2
```

Comment:
v2 is faster. Looks like its 2s vs 25s to checkout the repo. Small things add up :)